packages/buttons/src/elements/ToggleIconButton.tsx

```diff
@@ -11,9 +11,8 @@ import IconButton, { IIconButtonProps } from './IconButton';
 
 export interface IToggleIconButtonProps extends IIconButtonProps {
   /**
-   * Determine if the icon button is pressed or not. Use `'mixed'` to indicate
-   * whether the toggle controls other elements which do not share the same
-   * value.
+   * Determines if the icon button is pressed.
+   * Use `'mixed'` to indicate if the toggle controls other elements which don't share the same value.
```

Comment:
Paragraph formatting, as mentioned previously.

Reply:
Fixed in latest update